File: 프로그래머스/unrated/120912. 7의 개수/7의 개수.java
class Solution {
    public int solution(int[] array) {
        int answer = 0;
        
        for(int i = 0; i < array.length; i++) {
            String ele = Integer.toString(array[i]);
            String[] earr = ele.split("");
            // array의 원소를 문자로 변환하여 쪼갠뒤 earr로 배열화한다 
            for(int j = 0; j < earr.length; j++) {
                if(earr[j].equals("7")) {
                    answer++;
                } // earr의 원소 중 7이 있으면 answer를 증가시킨다
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/120912. 7의 개수/7의 개수.js
function solution(array) {
    let answer = 0;

    for (let i = 0; i < array.length; i++) {
        let ele = array[i].toString();
        let earr = ele.split("");
        // 배열의 요소를 문자열로 변환하여 분리한 후 배열로 저장합니다.
        for (let j = 0; j < earr.length; j++) {
            if (earr[j] === "7") {
                answer++;
            } // 배열의 요소 중 "7"이 있으면 answer를 증가시킵니다.
        }
    }
    return answer;
}
